packages/x-components/src/design-system/components/suggestion-group/default.tokens.scss

```diff
@@ -3,6 +3,7 @@
   --x-color-text-suggestion-group-default: var(--x-color-text-suggestion-default);
   --x-color-text-suggestion-group-matching-part-default: var(--x-color-text-suggestion-matching-part-default);
   --x-color-background-suggestion-group-default: var(--x-color-background-suggestion-default);
+  --x-color-border-suggestion-group-default: var(--x-color-text-default);
```

Comment:
better use the color of the text of the suggestion: `--x-color-text-suggestion-group-default`